src/js/controller/model.js

```diff
@@ -40,11 +40,10 @@ define([
                 buffer: 0
             });
 
-            // Mobile doesn't support autostart
-            // This check should be replaced with something that detects whether the current system
-            // requires a user interaction to start playback
-            if (utils.isMobile() && !config.mobileSdk) {
-                this.set('autostart', false);
+            // Videos autostart on mobile when the `muted` and `autoplay` attributes are set.
+            // Mute the player on a mobile device if autostart is set to true
+            if (this.get('autostart') && utils.isMobile() && !this.get('mobileSdk')) {
```

Comment:
Might want to movie this into its own function, so the logic is shared between here and below.